ONSdigital/sixteens
# Added COVID-19 banner colours

### What
- added new colour for the covid-19 banner
- tidied up the colours file

Colours file was misleading having colours listed with properties:
`name`, `primary`, `secondary` and `tertiary` when neither of the colours are primary, secondary or tertiary colours. But used as:
`name`, `background-colour`, `font-colour` and `anchor-colour`

### How to review

- Check the colour name
- Check the colours file is indeed better

### Who can review

Anyone except me
